.github/actions/checkout/action.yml

```diff
@@ -0,0 +1,154 @@
+name: 'Checkout'
+description: 'Checks out Electron and stores it in the AKS Cache'
+inputs:
+  generate-sas-token:
+    description: 'Whether to generate and persist a SAS token for the item in the cache'
+    required: false
+    default: 'false'
```

Comment:
This should be a boolean - the issues i think you were probably having were bc you aren't specifying a type here and it defaults to string otherwise